README.md

```diff
@@ -1 +1,148 @@
-# Hello-Microverse
\ No newline at end of file
+<a name="readme-top"></a>
+
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Hello Microverse Project] <a name="about-project"></a>
+
+
+
+**[Hello Microverse Project]** This is a simple project that displays "hello microverse".Its a true reflection of skills I have acquired from microverse so far.
+## 🛠 Built With <a name="built-with"></a>
+- HTML.
+- CSS.
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
```

Comment:
- [ ] While you make a great effort 😃  I see that you forgot to add the key features section here you know the importance of the key features to describe the advantages of the project and the features you used in it, so please try to add it and please 😊 try to not remove any section from the Readme as all sections are necessary the only optional section you can remove it is the FAQ section 💯🚀